ambari-server/src/main/resources/common-services/AMBARI_INFRA_SOLR/0.1.0/package/scripts/command_commons.py

```diff
@@ -94,7 +95,7 @@ def create_solr_api_request_command(request_path, output=None):
   solr_url = format("{solr_base_url}/{request_path}")
   grep_cmd = " | grep 'solr_rs_status: 200'"
   api_cmd = format("kinit -kt {keytab} {principal} && curl -w'solr_rs_status: %{{http_code}}' -k --negotiate -u : '{solr_url}'") \
```

Comment:
what will be the value of _keytab_, _principle_ and _api_cmd_ if security is not enabled?

Reply:
it's true that is not too readable with that linebreak, but in the next line there is that if params.security_enabled check, that means keytab and principal variableswont be referenced.

Reply:
ok, a missed that